Source/WebCore/css/parser/CSSPropertyParserConsumer+Timeline.cpp

```diff
@@ -150,5 +152,43 @@ RefPtr<CSSValue> consumeViewTimelineInset(CSSParserTokenRange& range, const CSSP
     });
 }
 
+static bool isAnimationRangeKeyword(CSSValueID id)
+{
+    return identMatches<CSSValueNormal, CSSValueCover, CSSValueContain, CSSValueEntry, CSSValueExit, CSSValueEntryCrossing, CSSValueExitCrossing>(id);
+}
+
+RefPtr<CSSValue> consumeAnimationRange(CSSParserTokenRange& range, const CSSParserContext& context, TimelineRangeStartOrEnd isStart)
+{
+    // https://drafts.csswg.org/scroll-animations-1/#propdef-animation-range-start
+    // normal | <length-percentage> | <timeline-range-name> <length-percentage>?
+    // TODO: handle animation range sequences
+
+    auto id = range.peek().id();
+
+    if (id == CSSValueNormal)
```

Comment:
This is unnecessarily inconsistent. We already consume an identifier after this. Why not check for `normal` there instead of peeking first?

Reply:
Fixed.